config.yml.default

```diff
@@ -1,131 +1,131 @@
+  name: "engine_name"              # Binary name of the engine to use.
+  working_dir: ""                  # Directory where the chess engine will read and write files. If blank or missing, the current directory is used.
+                                   # NOTE: If working_dir is set, the engine will look for files and directories relative to this directory, not where lichess-bot was launched. Absolute paths are unaffected.
+  protocol: "uci"                  # "uci", "xboard" or "homemade"
+  ponder: true                     # Think on opponent's time.
 
   polyglot:
-    enabled: false           # Activate polyglot book.
+    enabled: false                 # Activate polyglot book.
     book:
-      standard:              # List of book file paths for variant standard.
+      standard:                    # List of book file paths for variant standard.
         - engines/book1.bin
         - engines/book2.bin
-#     atomic:                # List of book file paths for variant atomic.
+#     atomic:                      # List of book file paths for variant atomic.
 #       - engines/atomicbook1.bin
 #       - engines/atomicbook2.bin
 #     etc.
 #     Use the same pattern for 'giveaway' (antichess), 'crazyhouse', 'horde', 'kingofthehill', 'racingkings' and '3check' as well.
-    min_weight: 1            # Does not select moves with weight below min_weight (min 0, max: 65535).
-    selection: "weighted_random" # Move selection is one of "weighted_random", "uniform_random" or "best_move" (but not below the min_weight in the 2nd and 3rd case).
-    max_depth: 8             # How many moves from the start to take from the book.
+    min_weight: 1                  # Does not select moves with weight below min_weight (min 0, max: 65535).
+    selection: "best_move" # Move selection is one of "weighted_random", "uniform_random" or "best_move" (but not below the min_weight in the 2nd and 3rd case).
+#   max_depth: 8                   # How many moves from the start to take from the book.
 
   draw_or_resign:
-    resign_enabled: false
-    resign_score: -1000      # If the score is less than or equal to this value, the bot resigns (in cp).
+    resign_enabled: false          # Whether or not the bot should resign.
+    resign_score: -1000            # If the score is less than or equal to this value, the bot resigns (in cp).
     resign_for_egtb_minus_two: true # If true the bot will resign in positions where the online_egtb returns a wdl of -2.
-    resign_moves: 3          # How many moves in a row the score has to be below the resign value.
-    offer_draw_enabled: false
-    offer_draw_score: 0      # If the absolute value of the score is less than or equal to this value, the bot offers/accepts draw (in cp).
+    resign_moves: 3                # How many moves in a row the score has to be below the resign value.
+    offer_draw_enabled: true       # Whether or not the bot should offer/accept draw.
```

Comment:
I don't have a problem having this enabled by default.

Reply:
Okay.